Tools/python/CondorSubmitter.py

```diff
@@ -7,19 +7,26 @@
 from JMTucker.Tools.CondorTools import cs_timestamp
 from JMTucker.Tools.general import mkdirs_if_needed, popen, save_git_status, int_ceil, touch
 
+#+SingularityImage = "/cvmfs/unpacked.cern.ch/registry.hub.docker.com/cmssw/el7:x86_64" #Alec added
+
 if not os.environ.has_key('SCRAM_ARCH') or not os.environ.has_key('CMSSW_VERSION'):
     raise EnvironmentError('CMSSW environment not set?')
 
+#Alec added: export X509_CERT_DIR=/cvmfs/grid.cern.ch/etc/grid-security/certificates/ 11 lines below
 class CondorSubmitter:
     sh_template = '''#!/bin/bash
 
+export X509_CERT_DIR=/cvmfs/grid.cern.ch/etc/grid-security/certificates/
+
 workdir=$(pwd)
```

Comment:
@duquale We can probably drop these three things, right? The X509 part comes in again a few lines later, so I think it's extraneous here.